<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>425</x>
         <y>185</y>
         <width>430</width>
         <height>350</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">.QFrame{
background-color: rgb(40, 40, 40);
border-radius: 10px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>20</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="assets.qrc">:/Ressources/Images/babel.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLineEdit{
border:2px solid rgb(18, 18, 18);
color:rgb(208, 208, 208);
background:rgb(50, 50, 50)
}</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
        <property name="placeholderText">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>210</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLineEdit{
border:2px solid rgb(18, 18, 18);
color:rgb(208, 208, 208);
background:rgb(50, 50, 50);
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Pasword</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLabel{
color:rgb(138, 138, 138)
}</string>
        </property>
        <property name="text">
         <string>Nom d'utilisateur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLabel{
color:rgb(138, 138, 138)
}</string>
        </property>
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>270</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">.QPushButton {
border-radius: 5px;
background:rgb(152, 0, 0);
color:rgb(208, 208, 208);
}

.QPushButton:pressed {
background:rgb(89, 89, 89);
}</string>
        </property>
        <property name="text">
         <string>Se connecter</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="assets.qrc">:/Ressources/Images/20860.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{
border:none;
background-color:rgb(62, 62, 62);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>label_5</zorder>
     <zorder>frame_2</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{
background-color:rgb(52, 52, 52);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{
background-color:rgb(33, 33, 33);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.Line{
border-color:rgb(117, 117, 117);
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>670</y>
        <width>1001</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QTextEdit{
border:none;
background-color:rgb(48, 48, 48);
color:rgb(255, 255, 255);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>-131</y>
        <width>371</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{
background-color:rgba(38, 38, 38, 150)
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>221</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QListWidget{
background-color:rgb(33, 33, 33);
border:none;
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pseudo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel
{
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Pseudo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="status">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>99</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{
background-color:rgb(0, 255, 0);
border-radius:10px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="call" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>0</y>
        <width>241</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="receive" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>190</y>
        <width>260</width>
        <height>325</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel{
border-radius:20px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="assets.qrc">:/Ressources/Images/if_User1_34247.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deco">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>680</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton{
background-color:rgb(103, 9, 9);
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Déconnexion</string>
      </property>
     </widget>
     <widget class="incall" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>400</width>
        <height>133</height>
       </rect>
      </property>
     </widget>
     <zorder>label_6</zorder>
     <zorder>label</zorder>
     <zorder>line</zorder>
     <zorder>textEdit</zorder>
     <zorder>label_8</zorder>
     <zorder>listWidget</zorder>
     <zorder>pseudo</zorder>
     <zorder>widget</zorder>
     <zorder>widget_2</zorder>
     <zorder>label_9</zorder>
     <zorder>status</zorder>
     <zorder>deco</zorder>
     <zorder>widget_3</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>call</class>
   <extends>QWidget</extends>
   <header>call.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>receive</class>
   <extends>QWidget</extends>
   <header>receive.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>incall</class>
   <extends>QWidget</extends>
   <header>incall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
